coa-helm/templates/tes-deployment.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    io.kompose.service: tes
+  name: tes
```

Comment:
I couldn't make an educated guess as to what this container is doing

Reply:
I'm not surprised :) This container runs the [TES](https://github.com/ga4gh/task-execution-schemas) server, which accepts tasks from Cromwell and runs them on Azure Batch. We'll talk about this in more detail at our meeting on Friday.